lib/handlers/injector-sor.ts

```diff
@@ -240,11 +242,23 @@ export abstract class InjectorSOR<Router, QueryParams> extends Injector<
             break
         }
 
-        const v3PoolProvider = new CachingV3PoolProvider(
+        const sourceOfTruthV3PoolProvider = new V3PoolProvider(chainId, multicall2Provider)
+        const currentV3PoolProvider = new CachingV3PoolProvider(
           chainId,
-          new V3PoolProvider(chainId, multicall2Provider),
+          sourceOfTruthV3PoolProvider,
           new NodeJSCache(new NodeCache({ stdTTL: 180, useClones: false }))
         )
+        const targetV3PoolProvider = new DynamoDBCachingV3PoolProvider(
```

Comment:
let's also rename this as `dynamoV3PoolProvider` or `awsCachingV3PoolProvider` or something like that